modules/rubiconBidAdapter.js

```diff
@@ -313,6 +317,7 @@ export const spec = {
       'p_pos',
       'gdpr',
       'gdpr_consent',
+      'schain',
```

Comment:
Sorry - just found out this needs to be `rp_schain`, not `schain`.

Reply:
Done